### ISSUE TITLE
TalkBack doesn't announce invalid radio buttons

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#2580]
# 🐛 Bug Report 

TalkBack doesn't announce radio buttons within a radio group w/ aria-invalid="true" applied to it. https://a11ysupport.io/tech/aria/aria-invalid_attribute seems to indicate that this is not just limited to inputs of type radio for TalkBack.

## 🤔 Expected Behavior

TalkBack should announce radio buttons as invalid

## 😯 Current Behavior

TalkBack doesn't announce radio buttons as invalid

## 💁 Possible Solution

May not be one, look to see if there is a bug filed against TalkBack. If no bug exists, file one.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3.14.1
| Browser          |
| Operating System |

## 🧢 Your Company/Team

RSP
